tests/trestle/tasks/xlsx_to_oscal_profile_test.py

```diff
@@ -81,6 +81,86 @@ def test_xlsx_execute(tmp_path):
         assert (result)
 
 
+@patch(target='uuid.uuid4', new=uuid_mock1)
+@patch(target='trestle.tasks.xlsx_to_oscal_profile.get_trestle_version', new=get_trestle_version_mock1)
+def test_xlsx_execute_with_blank_rows(tmp_path):
+    """Test execute call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path(CONFIG_PATH)
+    config.read(config_path)
+    section = config['task.xlsx-to-oscal-profile']
+    d_expected = pathlib.Path(section['output-dir'])
+    d_produced = tmp_path
+    section['output-dir'] = str(tmp_path)
+    section['spread-sheet-file'] = 'tests/data/spread-sheet/good_with_blank_rows.xlsx'
+    tgt = xlsx_to_oscal_profile.XlsxToOscalProfile(section)
+    tgt.set_timestamp('2021-07-19T14:03:03.000+00:00')
+    retval = tgt.execute()
+    assert retval == TaskOutcome.SUCCESS
+    list_dir = os.listdir(d_produced)
+    assert len(list_dir) == 1
+    assert d_expected != d_produced
+    for fn in list_dir:
+        f_expected = d_expected / fn
+        f_produced = d_produced / fn
+        result = text_files_equal(f_expected, f_produced)
+        assert (result)
+
```

Comment:
I don't think there is a need for parens on result - and it looks like you do it for most of your tests.  Ideally they would all be changed simply to `assert result` but I guess it isn't critical.

Reply:
I opened #1101 and will fix all with a subsequent PR sometime after trestle 1.0.0.